### PR TITLE
User can add songs

### PR DESCRIPTION
**Pull request by BHamp**
# Issue
As a Musician I want to store my transcriptions of Songs so I can refer to them later online

# Changes
Introduces song model, a form to create them, and a page to view them

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  get 'songs/new'
```
[the remark this answers is not in the document]
Comment (spilth) [reply config/routes.rb]: I removed this line for you

**Review by spilth — APPROVED** [review]
I added one fix to the branch.

Looks good for merging 👍

File: app/views/songs/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h1>New Song</h1>
+<%= form_with model: @song do |form| %>
+<%= form.label :title %>
```
[the remark this answers is not in the document]
Comment (BHamp) [reply app/views/songs/new.html.erb]: Fixed

File: spec/views/songs/new.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
[the remark this answers is not in the document]
Comment (BHamp) [reply spec/views/songs/new.html.erb_spec.rb]: I went ahead and cleaned this and the other empty files up

**Comment by spilth** [comment]
When I check out the latest version of this branch and run `rake`, I get the following errors:

```
An error occurred while loading ./spec/helpers/songs_helper_spec.rb.
Failure/Error:
  RSpec.describe SongsHelper, type: :helper do
    pending "add some examples to (or delete) #{__FILE__}"
  end

NameError:
  uninitialized constant SongsHelper
# ./spec/helpers/songs_helper_spec.rb:13:in `<main>'

An error occurred while loading ./spec/models/song_spec.rb.
Failure/Error:
  RSpec.describe Song, type: :model do
    pending "add some examples to (or delete) #{__FILE__}"
  end

NameError:
  uninitialized constant Song
# ./spec/models/song_spec.rb:3:in `<main>'

Finished in 0.00011 seconds (files took 7.98 seconds to load)
0 examples, 0 failures, 2 errors occurred outside of examples
```

It looks like you removed implementation files but not the specs referencing them.

File: app/models/song.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+class Song < ApplicationRecord
```
[the remark this answers is not in the document]
Comment (BHamp) [reply app/models/song.rb:1]: I did need to implement this file, so put it back in

[the remark this answers is not in the document]
Comment (spilth) [reply app/models/song.rb:1]: I shouldn't have thumbed up this. We definitely need this file!

**Review by spilth — APPROVED** [review]
Looks good. Thanks for taking care of the feedback!

**Pull request merged**